recipes/configure.rb

```diff
@@ -4,6 +4,46 @@
 #
 # Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
 
+#
```

Comment:
Should this be in it's own recipe?

Reply:
Put it back where it was and set the map key/value in the docker recipe (2cdb85f).